### ISSUE TITLE
How can I use in_range?

### ISSUE THREAD

**Issue by aemino** [twistedfall/opencv-rust#61]
I'm trying to use `opencv::core::in_range`, like such...
```rust
let mut mask = Mat::default().unwrap();
opencv::core::in_range(&img, &lower_bound, &upper_bound, &mut mask).unwrap();
```
However, I would like `lower_bound` and `upper_bound` to be color values. Looking at the OpenCV documentation, it looks like I should pass in `Scalar`s for these bounds. However, `Scalar` doesn't conform to `ToInputArray`, so I'm not sure how I can accomplish this.

**Comment by twistedfall** [comment]
I’ll take a look at it tomorrow, but for now you should be able to use 1x4 or 4x1 `Mat` with type `f64` instead of `Scalar`.

**Comment by aemino** [comment]
I have already tried this:
```rust
let lower_bound = Mat::from_slice(&[0.0, 0.0, 0.0, 0.0]).unwrap();
let upper_bound = Mat::from_slice(&[0.0, 0.0, 0.0, 0.0]).unwrap();
let mut mask = Mat::default().unwrap();
opencv::core::in_range(&img, &lower_bound, &upper_bound, &mut mask).unwrap();
```
which raises the following error:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error { code: -209, message: "OpenCV(4.1.1) C:\\build\\master_winpack-build-win64-vc14\\opencv\\modules\\core\\src\\arithm.cpp:1742: error: (-209:Sizes of input arguments do not match) The lower boundary is neither an array of the same size and same type as src, nor a scalar in function \'cv::inRange\'\n" }', src\libcore\result.rs:1165:5
```

**Comment by twistedfall** [comment]
Fixed in 0.25.0, you can now use `Scalar` as `InputArray`
